db/migrate/20160413171549_unique_roles.rb

```diff
@@ -0,0 +1,14 @@
+class UniqueRoles < ActiveRecord::Migration
+  def up
+    UserProjectRole.all.group_by(&:user_id).each do |user, roles|
+      roles.sort_by(&:role_id)[0..-2].each(&:destroy)
+    end
```

Comment:
Wary of doing migrations that depend on ActiveRecord models.  If we nuke that class 6 months from now, running `rake db:migrate` on a new DB will fail.